File: smallerNumbersThanCurrent.js
const smallerNumbersThanCurrent = function(nums) {
  let counter = 0
  const arrHolder = [];
    for(let i=0; i<nums.length; i++){
      
      for(let j=0; j<nums.length; j++){
        if(nums[i]>nums[j]){
          counter++;
        }
      }
      arrHolder.push(counter);
      counter = 0;
    }
    return arrHolder;
};

File: smallerNumbersThanCurrent.py
class Solution(object):
    def smallerNumbersThanCurrent(self, nums):
        counter = []
        for i in nums:
            counter.append(0)

        for i in range(len(nums)):
            for j in range(len(nums)):
                if (nums[i] > nums[j]):
                    counter[i] = counter[i] + 1

        return counter
